snuba/web/rpc/v1/resolvers/common/aggregation.py

```diff
@@ -29,12 +29,17 @@
 from snuba.web.rpc.common.exceptions import BadSnubaRPCRequestException
 
 sampling_weight_column = column("sampling_weight")
+sampling_factor_column = column("sampling_factor")
 
 Z_VALUE_P95 = 1.96  # Z value for 95% confidence interval is 1.96 which comes from the normal distribution z score.
 Z_VALUE_P975 = 2.24  # Z value for 97.5% confidence interval used for the avg() CI
 
 PERCENTILE_PRECISION = 100000
 PERCENTILE_SAMPLE_COUNT_THRESHOLD = 50
+PERCENTILE_CORRECTION_FACTOR = (
+    1000  # We multiply the sampling weight by this factor to reduce rounding error
```

Comment:
could use more detailed explanation here as to why. Your document has good context you can include here